src/vos/vos_internal.h

```diff
@@ -353,6 +353,14 @@ struct vos_container {
 	 * * transaction with older epoch must have been committed.
 	 */
 	daos_epoch_t		vc_solo_dtx_epoch;
+
+	/* The lowest active epoch and the highest committed epoch will
+	 * be used as the start epoch during incremental reintegration,
+	 * so during reintegration, vos_cont_open and dtx_cont_open will
+	 * update these 2 epochs.
+	 */
+	daos_epoch_t		vc_lowest_act_eph;
+	daos_epoch_t		vc_highest_cmt_eph;
```

Comment:
Sorry, where are these new members used except being assigned as some value? I do not see where they are returned back to upper layer in the patch.

Reply:
ah, they will used by incremental reintegration, and this is for delay rebuild only. So they will be used later once incremental reintegration is landed, https://github.com/daos-stack/daos/pull/13358.  Though this is targeted for 2.8

Reply:
https://github.com/daos-stack/daos/pull/13879  to refine the issue